# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/svg/crashtests/1848851.html
# Commit: 78e7cb2837e7
# Full Hash: 78e7cb2837e7dec0ef55a2f5f383625335ad2520
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-08-17 15:01:15
# Description:
#   Bug 1848851 - Check target element is valid type before notifying r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186358
# ==============================================================================

diff -r 970525aa04f1 -r 78e7cb2837e7 layout/svg/crashtests/1848851.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/svg/crashtests/1848851.html	Thu Aug 17 10:06:16 2023 +0000
@@ -0,0 +1,13 @@
+<script>
+document.addEventListener("DOMContentLoaded", () => {
+  b.text = "-a"
+  c.scrollLeft
+  c.appendChild(b)
+})
+</script>
+<svg>
+<text direction="rtl">
+<textPath xlink:href="#a">
+<tspan id="a">
+<a id="b" />
+<feDistantLight id="c" />